menu.html

```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>OUR MENU</h1>
+    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Harum incidunt odio non, quisquam vitae, perspiciatis commodi suscipit, vel doloribus corporis vero rerum! Dolorem debitis voluptas libero aspernatur commodi vel quis?</p>
```

Comment:
change the line to something more tasty like `<p>Lorem awesome food menu! Yum!</p>`